### PR TITLE
Implement tensorflow for comparison

### PR DESCRIPTION
**Pull request by plpxsk**
Check things like loss function, and compare final weights

### Resources

https://www.tensorflow.org/tfmodels/nlp/fine_tune_bert

### CODE REVIEW

**Comment by plpxsk** [comment]
In tf_qa nb, compare losses between mlx and tf. Some dimension issue encountered

WIP
